arangod/Cluster/ClusterMethods.cpp

```diff
@@ -4053,16 +4053,17 @@ arangodb::Result hotBackupCoordinator(VPackSlice const payload, VPackBuilder& re
 
     bool force = !payload.isNone() && payload.get("forceBackup").isTrue();
 
-    std::string const backupId =
-      (payload.isObject() && payload.hasKey("label")) ?
-      payload.get("label").copyString() : to_string(boost::uuids::random_generator()());
+    std::string const backupId = (payload.isObject() && payload.hasKey("label"))
```

Comment:
Why don't you use velocypackHelper functions for this?
No need to do ternary with 3 checks every time, it has helper functions for `give me value or if unable the default`